Testscripts/Linux/perf_fio_nfs.sh

```diff
@@ -177,6 +177,12 @@ if [ $? -eq 0 ]; then
 
 	mountDir="/data"
 	cd ${HOMEDIR}
+	if [[ $DISTRO == "redhat_7" ]]; then
+		ssh root@nfs-server-vm "systemctl stop firewalld"
+		ssh root@nfs-server-vm "systemctl disable firewalld"
+		systemctl stop firewalld
+		systemctl disable firewalld
+	fi
```

Comment:
We can use this function https://github.com/LiliDeng/LISAv2/blob/fix_issues/Testscripts/Linux/utils.sh#L3047, not do something special on RHEL7 only.

Reply:
This function is only working in one VM case. This needs both VM sizes.

Reply:
Do we need to check exit status for disabling firewall?

Reply:
Yes, we can, but if firewall setting fails, the test fails.

Reply:
does it captures cause of failure ?

Reply:
Fixed in abfd995810719d48db97227e1704aee0d9b25780